hypershift-operator/controllers/hostedcluster/hostedcluster_webhook.go

```diff
@@ -221,20 +252,48 @@ func (v nodePoolValidator) validateCreateKubevirtNodePool(ctx context.Context, n
 		return nil, fmt.Errorf("failed to retrive HostedCluster %s/%s; %w", np.Namespace, np.Spec.ClusterName, err)
 	}
 
-	return kvValidator.validate(ctx, v.client, hc)
+	return kvValidator.validate(ctx, hc, np.Spec.Release.Image)
+}
+
+func (v nodePoolValidator) validateUpdateKubevirtNodePool(ctx context.Context, oldNP, newNP *hyperv1.NodePool) error {
+	err := validateJsonAnnotation(newNP.Annotations)
+	if err != nil {
+		return err
+	}
+
+	if oldNP.Spec.Release.Image != newNP.Spec.Release.Image {
+		hc := &hyperv1.HostedCluster{}
+		err = v.client.Get(ctx, client.ObjectKey{Name: newNP.Spec.ClusterName, Namespace: newNP.Namespace}, hc)
+		if err != nil {
+			return fmt.Errorf("failed to retrive HostedCluster %s/%s; %w", newNP.Namespace, newNP.Spec.ClusterName, err)
+		}
```

Comment:
I think we should ignore this release image validation on the nodepool in the event that the HostedCluster cannot be found for some reason.

technically, that scenario shouldn't happen, but if it does, i think it would be best if the validation webhook didn't block the nodepool.